web-client/src/presenter/actions/FileDocument/refreshExternalDocumentTitleFromEventCodeAction.test.js

```diff
@@ -101,25 +89,59 @@ describe('refreshExternalDocumentTitleFromEventCodeAction', () => {
   });
 
   it('should not overwrite document title when the eventCode is included in PRACTITIONER_ASSOCIATION_DOCUMENT_TYPES_MAP', async () => {
-    const entryOfAppearanceDocument = {
-      category: 'Appearance and Representation',
-      documentTitle: 'Entry of Appearance for Petitioner Queen Clarion',
-      documentType: 'Entry of Appearance',
-      eventCode: 'EA',
-      scenario: 'Standard',
-    };
+    const mockEntryOfAppearanceDocumentTitle =
+      'Entry of Appearance for Petitioner Queen Clarion';
+
     const result = await runAction(
       refreshExternalDocumentTitleFromEventCodeAction,
       {
         modules: { presenter },
         state: {
-          form: entryOfAppearanceDocument,
+          form: {
+            category: 'Appearance and Representation',
+            documentTitle: mockEntryOfAppearanceDocumentTitle,
+            eventCode: 'EA',
+          },
         },
       },
     );
 
     expect(result.state.form.documentTitle).toEqual(
-      entryOfAppearanceDocument.documentTitle,
+      mockEntryOfAppearanceDocumentTitle,
+    );
+  });
+
+  it('should set the document title when the eventCode is an internal filing event code', async () => {
+    const result = await runAction(
+      refreshExternalDocumentTitleFromEventCodeAction,
+      {
+        modules: { presenter },
+        state: {
+          form: {
+            category: 'Miscellaneous',
+            eventCode: 'MISC',
+          },
+        },
+      },
+    );
+
+    expect(result.state.form.documentTitle).toEqual('[Miscellaneous]');
+  });
+
+  it('should set the secondary document title when the eventCode is an internal filing event code', async () => {
+    const result = await runAction(
+      refreshExternalDocumentTitleFromEventCodeAction,
+      {
+        modules: { presenter },
+        state: {
+          form: {
+            category: 'Miscellaneous',
+            eventCode: 'MISC',
+          },
+        },
+      },
     );
+
```

Comment:
Hmm these last two tests are the same, but have different names. I think we should test for the secondary document.